packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -253,7 +252,7 @@ function GridFilterForm(props: GridFilterFormProps) {
           ))}
         </Select>
       </FormControl>
-      <FormControl variant="standard" sx={{ width: 190 }}>
+      <FormControl variant="standard" sx={{ minWidth: 190 }}>
```

Comment:
I'm not sure if removing the width is correct, maybe we should set `max-width` too. cc @oliviertassinari @danilo-leal 

![image](https://user-images.githubusercontent.com/42154031/146984641-279db85c-93c5-4ac6-87ad-76e486cb726e.png)